src/Link.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Bootstrap5;
+
+final class Link
```

Comment:
`Link` without URL is not link. May be call it "BreadcrumbItem" or "Breadcrumb" ?

Reply:
https://github.com/yiisoft/yii-bootstrap5/blob/c750b36367f5a65503ebc4abbea2b79dcc805f38/src/Link.php#L13 It may or may not have a URL, so it is correct to call it Link.